lib/galaxy/webapps/galaxy/controllers/dataset.py

```diff
@@ -18,6 +18,7 @@
     web
 )
 from galaxy.datatypes.display_applications.util import decode_dataset_user, encode_dataset_user
+from galaxy.datatypes import sniff
```

Comment:
This needs to be moved one line above, see our import order style: https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_smarkets.py